rfc-template.md

```diff
@@ -0,0 +1,24 @@
+# Project title
```

Comment:
I would love to be explicit in the template that no fields are required when filling this out and submitting it. or something like that.

Reply:
👍 will change